pkg/cli/mirror/copy.go

```diff
@@ -612,44 +612,38 @@ func getRelatedImages(directory string, packages []v1alpha2.IncludePackage) ([]d
 	return finalList, nil
 }
 
-func isPackageSelected(bundle declcfg.Bundle, channels []declcfg.Channel, packages []v1alpha2.IncludePackage) (bool, error) {
-	isSelected := false
-	for _, pkg := range packages {
-		if pkg.Name == bundle.Package {
-			var min, max semver.Version
-			if pkg.MinVersion != "" || pkg.MaxVersion != "" {
-				version_string, err := bundleVersion(bundle)
+func isBundleSelected(bundle declcfg.Bundle, catalogChannels []declcfg.Channel, catalogFilters []v1alpha2.IncludePackage) (bool, error) {
```

Comment:
Could you include an imagesetconfig you used  to verify this , as well as command line with args. I want to verify locally

Reply:
Hi @lmzuccarelli 

As requested, imagesetconfig below

```
kind: ImageSetConfiguration
apiVersion: mirror.openshift.io/v1alpha2
storageConfig:
  local:
    path: ./alex_local_storage
mirror:
  operators:
  - catalog: registry.redhat.io/redhat/redhat-operator-index:v4.11
    packages:
    - name: aws-load-balancer-operator
      #minVersion: 0.0.0
      #maxVersion: 0.1.9
      channels:
      - name: stable-v0
        minVersion: 0.0.0
        maxVersion: 0.3.0
```

And the command used during my tests locally:

`./bin/oc-mirror --use-oci-feature --oci-feature-action=copy oci://alex_tests --config imstcfg.yaml`